### ISSUE TITLE
Lua Error when you Try to Use Items in Tracker while in Combat

### ISSUE THREAD

**Issue by Gogo1951** [Questie/Questie#4628]
2x [ADDON_ACTION_FORBIDDEN] AddOn 'Questie' tried to call the protected function 'UseItemByName()'.
[string "@Interface\AddOns\!BugGrabber\BugGrabber.lua"]:480: in function <Interface\AddOns\!BugGrabber\BugGrabber.lua:480>
[string "=[C]"]: in function `UseItemByName'
[string "@Interface\FrameXML\ChatFrame.lua"]:1150: in function `SecureCmdUseItem'
[string "@Interface\FrameXML\SecureTemplates.lua"]:412: in function `handler'
[string "@Interface\FrameXML\SecureTemplates.lua"]:625: in function <Interface\FrameXML\SecureTemplates.lua:569>

**Comment by Muehe** [comment]
If you have the time for testing, I have an inkling that reverting c00c8ffc547be529b25ca91b2441f3dba6f0a84a , specifically removing line 549, might help with this (but reintroduce the collapsing tracker bug).

**Comment by Gogo1951** [comment]
Yes, seems like reverting https://github.com/Questie/Questie/commit/c00c8ffc547be529b25ca91b2441f3dba6f0a84a makes this bug stop.

**Comment by Dyaxler** [comment]
I just tried to reproduce this and received no error... I actually remember this bug. It generally happens when the ItemButton is trying to be shown or hidden while in combat. My fix was to wrap this section of code in the CombatQue. Matter of fact, at one point the player could minimize the Quest but to minimize this issue I just hid the Min/Max button under the ItemButton unless the quest was complete. Then the ItemButton was hidden and the Min/Max button was visible.

I did however notice that while the option Minimize While In Combat was enabled, the tracker doesn't fully minimize. This fix has been checked into my Fork and should update with my [Pull Request.](https://github.com/Questie/Questie/pull/4664)
